src/common/system.rs

```diff
@@ -847,6 +819,39 @@ impl System {
         SystemInner::distribution_id_like()
     }
 
+    /// Provides kernel version following this string format:
+    ///
+    /// | Windows | Windows OS Build 20348.2227 |
```

Comment:
This needs a table header or the markdown won't render as a table.

<img src="https://github.com/user-attachments/assets/2ada445a-aa51-4b85-aa13-c371505c0052">

```suggestion
    /// | example platform | value of `System::kernel_long_version()` |
    /// |---|---|
    /// | Windows | Windows OS Build 20348.2227 |
```

Or write the table using html table tags.

Reply:
I was sure it could be skipped in markdown. Thanks for the heads up!